dracula-theme.el

```diff
@@ -56,7 +55,8 @@
    'dracula
    ;; default
    `(cursor ((,class (:background ,fg3))))
-   `(default ((,class (:background ,bg1 :foreground ,fg1))))
+   `(default ((((type nil)) (:background "#000000" :foreground ,fg1))
```

Comment:
I’m not sure I got your point about `default` though. The current frame being default applies to the invocation of `(window-system)` without arguments, which I suppose is what custom.el does, and it’s evaluated every time a new frame is created.